.github/workflows/labeler.yml

```
@@ -0,0 +1,17 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target:
```

actionlint (1.7.4)

expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)